glance/glance.go

```diff
@@ -106,3 +106,57 @@ func (c *Client) GetImageDetail(imageId string) (*ImageDetail, error) {
 	}
 	return &resp.Image, nil
 }
+
+// ImageDetailV2 describes extended information about an image for API v2.0.
```

Comment:
I know the upstream docs move around a  bit, but I'd like a url reference in the comment here so it's clearer what the various fields in the struct mean.